CMakelists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.21)
+list(APPEND CMAKE_PREFIX_PATH $CONDA_PREFIX)
+
+project(bitsandbytes LANGUAGES CXX CUDA)
+
+set(CXX_STANDARD_REQUIRED C++14)
+set(FILES_CUDA csrc/ops.cu csrc/kernels.cu)
+set(FILES_CPP csrc/common.cpp csrc/cpu_ops.cpp csrc/pythonInterface.c)
+
+option(MAKE_CUDA_BUILD "Build using CUDA" ON)
+option(NO_CUBLASLT "Don't use CUBLAST" OFF)
+set(CUDA_VERSION "11.7" CACHE STRING "CUDA Version DLL Name: 11.0, 11.7, 11.6")
```

Comment:
I need to see if it's better to use CMAKE_CUDA_COMPILER_VERSION to determine this variable. 

The CMAKE script can be run manually and have a specific CUDA version specified -D CUDA_TOOLKIT_ROOT_DIR....

Reply:
For windows - https://cmake.org/cmake/help/latest/variable/CMAKE_VS_PLATFORM_TOOLSET_CUDA.html